bigquery_etl/query_scheduling/templates/airflow_dag.j2

```diff
@@ -86,6 +86,9 @@ with DAG('{{ name }}', default_args=default_args{%+ if schedule_interval != None
             {%+ elif task.date_partition_parameter == None or task.date_partition_parameter is string -%}
             date_partition_parameter={{ task.date_partition_parameter | format_optional_string }},
             {%+ endif -%}
+            {%+ if task.table_partition_template -%}
+            table_partition_template='{{ task.table_partition_template }}',
+            {%+ endif -%}
```

Comment:
To avoid all these DAG changes, we could add a condition here to only add  `table_partition_template` if it doesn't match the default `{{ ds_nodash }}`